harmonica/forward/tesseroid_layer.py

```diff
@@ -97,6 +99,15 @@ def _check_regular_grid(longitude, latitude):
         raise ValueError("Passed latitude coordinates are note evenly spaced.")
 
 
+def _check_overlap(longitude):
+    """
+    Check if the prisms boundaries are overlapped
+    """
+    spasing = longitude[1] - longitude[0]
+    if longitude.max() - longitude.min() >= 360 - spasing:
+        raise ValueError("Tesseroid boundaries are overlapped.")
```

Comment:
Fix typo on `spacing`:

```suggestion
    spacing = longitude[1] - longitude[0]
    if longitude.max() - longitude.min() >= 360 - spacing:
        raise ValueError("Tesseroid boundaries are overlapped.")
```